webrtc.html

```diff
@@ -6570,9 +6594,22 @@ <h2>Attributes</h2>
               <code><a>RTCRtpReceiver</a></code> object <var>receiver</var>.
               When one of the SSRCs for RTP source media streams received
               by <var>receiver</var> is removed (either due to reception of
-              a BYE or via timeout), the <code>mute</code> event is fired at
-              <code>track</code>.  If and when packets are received again,
-              the <code>unmute</code> event is fired at <code>track</code>.</p>
+              a BYE or via timeout), <a>set the muted state</a> of the
+              <code>track</code> to the value <code>true</code>.
+              Note that <a data-lt="set the RTCSessionDescription">
+              setting an <code>RTCSessionDescription</code></a>
+              can also lead to <a data-lt="set the muted state"> setting
+              the muted state</a> of the <code>track</code> to the
+              value <code>true</code>.
+              If packets are received again, and the <code>track</code> is
+              muted, and either the [[\OfferedDirection]] or
+              [[\CurrentDirection]] slot of the 
```

Comment:
If `[[OfferedDirection]]` is non-null, then I think it must trump [[CurrentDirection]] or it won't work right.

Reply:
I think the proposed text actually works, if at least one of [[OfferedDirection]] and [[CurrentDirection]] includes "receive" incoming packets can lead to "unmute".